pkg/cmd/operator/cmd.go

```diff
@@ -3,6 +3,8 @@ package operator
 import (
 	"github.com/spf13/cobra"
 
+	_ "github.com/openshift/api/config/v1/zz_generated.crd-manifests"
+	_ "github.com/openshift/api/operator/v1/zz_generated.crd-manifests"
```

Comment:
@sanchezl you might know the answer - should this go in https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/dependencymagnet/doc.go instead?